ksqldb-engine/src/test/java/io/confluent/ksql/services/FakeKafkaConsumerGroupClient.java

```diff
@@ -0,0 +1,46 @@
+package io.confluent.ksql.services;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.TopicPartition;
+
+public class FakeKafkaConsumerGroupClient implements KafkaConsumerGroupClient {
+
+  private static final List<String> groups = ImmutableList.of("cg1", "cg2");
+
+  @Override
+  public List<String> listGroups() {
+    return groups;
+  }
+
+  @Override
+  public ConsumerGroupSummary describeConsumerGroup(String group) {
+    if (groups.contains(group)) {
+      Set<ConsumerSummary> instances = ImmutableSet.of(
+          new ConsumerSummary(group + "-1"),
+          new ConsumerSummary(group + "-2")
+      );
+      return new ConsumerGroupSummary(instances);
+    } else {
+      return null;
```

Comment:
don't return null.  The contract is to throw an exception, e.g. `KafkaResponseGetFailedException`, on an unknown group.  (Not that the contract is documented!).